Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Toggle.Switch/Classes/Drawing/RoundedRectangleF.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Drawing;
+using System.Drawing.Drawing2D;
+
+namespace Krypton.Toolkit.Suite.Extended.Toggle.Switch
+{
+    /// <summary>
+    /// Code from: https://github.com/aalitor/AltoControls/blob/on-development/AltoControls/Helpers/RoundedRectangleF.cs
+    /// </summary>
+    internal class RoundedRectangleF
```

Comment:
This should be in the Standard toolkit, as used by buttons etc.

Reply:
Will move.

Reply:
See latest 'todo' comment